packages/storybook/tsconfig.json

```
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "lib",
+    "target": "es5",
+    "module": "commonjs",
+    "lib": ["es5", "esnext", "dom"],
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "moduleResolution": "node",
+    "preserveConstEnums": true,
+    "types": ["jest", "react", "react-dom"],
```

This setting is just for which types should be automatically included globally (types such as `react` will be automatically pulled in when the package is imported). Here the only thing you might need is `webpack-env`, since this package doesn't use jest.
```suggestion
    "types": ["webpack-env"],
```

---

I do need `react` here. so i updated to only that. I don't see a need for `webpack-env` for now or future? please correct me if i'm wrong here.